src/main/java/pages/ProfilePage.java

```diff
@@ -9,41 +9,25 @@
 
 public class ProfilePage extends BasePage {
 
-
     @FindBy(xpath = "//h1")
     private WebElement replays;
-
     @FindBy(xpath = "//p[contains(@class,'notifier__notification-message') and contains(text(),'Tylko dla zalogowanych!')]")
     private WebElement noteForLogged;
 
-
     public ProfilePage(WebDriver driver) {
         super(driver);
         PageFactory.initElements(driver, this);
-
-        // caltchAssertion = new caltchAssertion(driver);
     }
 
     public void notificationIsDisplayed() {
         Assert.assertTrue(noteForLogged.isDisplayed());
-
     }
 
     public void replaysIsDisplayedInH1andButton() {
-        System.out.println(replays.getText());
```

Comment:
Usunąłem zbędne `System.out.println`, leśli zależy Ci na wydrukowaniu czegoś na konsoli podczas testu, to użyj Loggera (np Logback).